backend/src/test/java/com/redhat/cloud/notifications/routers/EndpointResourceTest.java

```diff
@@ -713,7 +713,7 @@ void addSlackEndpoint() {
             assertEquals("#notifications", channel);
 
             ep.getProperties(CamelProperties.class).getExtras().put("channel", "#updated");
-            ep.getProperties(CamelProperties.class).setUrl("https://updated.com");
+            ep.getProperties(CamelProperties.class).setUrl("https://redhat.com");
```

Comment:
Seems https://updated.com could no more be resolved, since, it generates as validation exception.